qiskit-superstaq/qiskit_superstaq/compiler_output.py

```diff
@@ -124,7 +124,7 @@ def read_json_aqt(  # pylint: disable=missing-param-doc
     json_dict: Dict[str, str],
     metadata_of_circuits: List[Dict[Any, Any]],
     circuits_is_list: bool,
-    num_eca_circuits: int = 0,
+    num_eca_circuits: Optional[int] = 0,
```

Comment:
nit: unsure, and perhaps redundant, but possibly updating docstring to include a description of the optional `num_eca_circuits` argument

Reply:
Done. (Also made parameter name consistent, using `num_equivalent_circuits` everywhere.)